ftest/webdriver/src/test/java/com/nuxeo/functionaltests/ITEventCreationTest.java

```diff
@@ -82,14 +82,14 @@ protected DocumentBasePage createTestEvent(DocumentBasePage page) throws IOExcep
     }
 
     @Test
+    @Repeat(issue = "NXP-21383: The text of 'betweenBanner' cannot be found before timeout.")
```

Comment:
> Do not use `@Repeat` for IT tests

I agree with you.

> better fix test with the waitUntil functions from Locator

In this commit, the method `Locator#waitForTextPresent` is used and it already waits for 30 seconds. So just removing the `@Repeat` annotation would be enough, or other "wait-until" functions to ensure the robustness?